matching-notification-resources/src/main/resources/resources/uicomponents/matchingNotification/matchesTable.js

```diff
@@ -486,7 +487,7 @@ var PhenoTips = (function (PhenoTips) {
             onCreate : function () {
                 $("panels-livetable-ajax-loader").show();
                 this._utils.clearHint('send-notifications-messages');
-                $('send-notifications-button').addClassName("disabled");
```

Comment:
I'm not sure why `send-notification-button` is implemented as a `<a>` element. IMO it should really be a button, and that would make many things easier:
* Jumping to the top of the document when pressing "Send notifications" would never happen, no need to stop the event to prevent it
* It can be `.disable()`d for real , not just styled as disabled, preventing any functionality occurring when there's no notification to send

Reply:
I was thinking, that in a way it might be useful to give a feedback to user when there is no emails selected and user still can click on the button and see this message as a hint on what to do to use this functionality. So thats why I left the button like this, but may be its not that needed.

Reply:
In my opinion, the fact that the button is disabled should be enough in this case. Also, this is the admin UI, not for broad usage, so we don't need that many explanations. Robust and easily maintainable code logic is more important IMO.